packages/hint-validate-summary/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "outDir": "dist",
+        "strict": true
+    },
+    "exclude": [
+        "dist",
+        "node_modules"
+    ],
+    "extends": "../../tsconfig.json",
+    "include": [
+        "src/**/*.ts",
+        "tests/**/*.ts"
+    ],
+    "references": [
+        {
+            "path": "../hint"
+        },
+        {
+            "path": "../utils-tests-helpers"
+        }
+    ]
+}
```

Comment:
You need to run `yarn update:references` from the root of the repo to update this file.